services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall
+from typing import List, Optional
+import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(
+        session_date: Optional[str] = None
+) -> List[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: Optional[datetime.datetime] = None,
+        movie_id: Optional[int] = None,
+        cinema_hall_id: Optional[int] = None
+) -> MovieSession:
+    movie_session = get_movie_session_by_id(session_id)
+    if show_time:
+        movie_session.show_time = show_time
+    if movie_id:
+        movie_session.movie_id = movie_id
+    if cinema_hall_id:
+        movie_session.cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
Why did you decide to set movie by id and cinema_hall by whole object ? Chose smth one ) 
Imho setting by id it is more clean so ->
`movie_sessions.cinema_hall_id = cinema_hall_id`